hexrd/WPPF.py

```diff
@@ -1243,11 +1248,11 @@ def _nm(x):
         return valWUnit('lp', 'length', x, 'nm')
 
     def __init__(self,
-        expt_spectrum=None,
-        params=None,
-        phases=None,
-        wavelength={'kalpha1':_nm(0.15406),'kalpha2':_nm(0.154443)},
-        bkgmethod='spline'):
+                 expt_spectrum=None,
+                 params=None,
+                 phases=None,
+                 wavelength={'kalpha1':_nm(0.15406),'kalpha2':_nm(0.154443)},
+                 bkgmethod={'spline':None}):
```

Comment:
FYI, setting dicts as default arguments falls under the category of using "mutable default arguments", which can lead to difficult to track bugs, because python stores copies of the default arguments. See [here](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) for an example.

We would typically do something like this instead:
```python
def my_func(wavelength=None, bkgmethod=None):
    if wavelength is None:
        wavelength = {'kalpha1':_nm(0.15406),'kalpha2':_nm(0.154443)}
    if bkgmethod is None:
        bkgmethod = {'spline':None}
```

Reply:
Got it. I can fix these.